test/integration/idle_timeout_integration_test.cc

```diff
@@ -489,8 +489,8 @@ TEST_P(IdleTimeoutIntegrationTest, RequestTimeoutUnconfiguredDoesNotTriggerOnBod
 }
 
 TEST_P(IdleTimeoutIntegrationTest, RequestTimeoutTriggersOnRawIncompleteRequestWithHeaders) {
-  // Omitting \r\n\r\n does not indicate incomplete request in HTTP2
-  if (downstreamProtocol() == Envoy::Http::CodecType::HTTP2) {
+  // Omitting \r\n\r\n does not indicate incomplete request in HTTP2/3
+  if (downstreamProtocol() >= Envoy::Http::CodecType::HTTP2) {
```

Comment:
How about `!= Envoy::Http::CodecType::HTTP1` and in the comment:

```
// Omitting \r\n\r\n only indicate incomplete request in HTTP1.
```

Here and elsewhere...